tests/src/test/java/org/eclipse/hono/tests/client/ClientTestBase.java

```diff
@@ -232,8 +233,15 @@ public void testSendingMessages(final TestContext ctx) throws Exception {
             setup.complete();
         }));
 
-        sender.setDispositionHandler((id, disposition) -> accepted.countDown());
+        sender.setDispositionHandler((id, disposition) -> {
+            accepted.countDown();
+            //TODO temp fix to the test, until we have a logic to resend released qos1 messages
```

Comment:
Yeah, I was thinking more about adding more semantics to the sender to do a better job of being sure that message was accepted for qos1, instead of just fire and forget. I'll do more in this area after we have this phase finished.